RecoEgamma/EgammaElectronProducers/python/gsfElectronSequence_cff.py

```diff
@@ -7,31 +7,14 @@
 #==============================================================================
 
 from RecoEgamma.EgammaElectronProducers.ecalDrivenGsfElectronCores_cfi import ecalDrivenGsfElectronCores
-from RecoEgamma.EgammaElectronProducers.gsfElectronCores_cfi import gsfElectronCores
 from RecoEgamma.EgammaElectronProducers.ecalDrivenGsfElectronCoresFromMultiCl_cff import ecalDrivenGsfElectronCoresFromMultiCl
 from RecoEgamma.EgammaElectronProducers.gsfElectrons_cfi import *
```

Comment:
This import can probably also get removed now

Reply:
It needs to stay, because this file also contains the `ecalDrivenGsfElectrons`. I would not bother renaming it now thought, because at some point there will be probably an autogenerated ecalDrivenGsfElectrons_cfi anyway.